app/router.js

```diff
@@ -52,6 +52,7 @@ Router.map(function () {
   this.route('category-slugs', { path: 'category_slugs' });
   this.route('team', { path: '/teams/:team_id' });
   this.route('policies');
+  this.route('security');
```

Comment:
this makes it crates.io/security, I'm wondering if crates.io/policies/security would be better?

Reply:
I'm fine with crates.io/policies/security; my thought was mostly "what URL might someone type in trying to find this information?" 

What do you think about having the canonical location for this be crates.io/policies/security but have crates.io/security be a redirect here?

Reply:
yeah, that sounds good to me. let me know if you need help setting that up :)

Reply:
I _think_ I got it working. Definitely review my ember!! 😅

Reply:
looks exactly right! :)